app/services/csv_upload_normalizer.rb

```diff
@@ -11,10 +11,14 @@ def initialize(path, druid_headers: %w[Druid druid])
     @druid_headers = druid_headers
   end
 
-  # Reads and normalizes CSV.
+  # Reads uploaded file (could be csv or Excel) and normalizes to CSV.
   # This includes handling BOM and druids without prefixes.
   def read
-    table = CSV.open(path, 'rb:bom|utf-8', headers: true).read
+    raise 'Unsupported upload file type' unless %w[.csv .ods .xls .xlsx].include? File.extname(path)
+
+    spreadsheet = Roo::Spreadsheet.open(path, { csv_options: { encoding: 'bom|utf-8' } }) # open the spreadsheet
+    table = CSV.parse(spreadsheet.to_csv, headers: true) # convert spreadsheet to CSV and then parse by the CSV library
+
     table.each do |row|
```

Comment:
this is the core change, just have Roo open the spreadsheet, using the CSV encoding (if you happen to open a CSV file to start), then ask roo to covert to CSV and parse it as normal

the desc metadata import job has it's form changed to accept other file types (and note this in the message to the user), the other forms do not even though they could now, but should continue to work exactly as before since CSV is an accepted type